### PR TITLE
Refactored HealthNHunger to player status and added more meters

### PR DESCRIPTION
**Pull request by BrailleBennett**
# Changelog
## New Features
- HealthNHunger is now player status
- Made the readout of player status more concise which also delivers values faster
- This includes new information like air, frost exposure, armor, and absorption hearts
- Added a warning for when frost exposure gets above a configurable value (default 30%)
- Hold Alt+R to hear only conditional information like air and frost exposure
- Renamed enableHealthNHunger in the config to enablePlayerStatus (**THIS IS A BREAKING CHANGE**)

## Feature Updates
- Air is now measured roughly in seconds remaining, as it is not clear how fast air ticks down for blind players usually
- The default air threshold is now 5 seconds of air remaining, which is roughly the same overall percentage (30%) air remaining

### CODE REVIEW

**Comment by BrailleBennett** [comment]
Another known issue is that the config seems to be broken and isn't filling in after it generates following me adding my frost exposure threshold option.

File: common/src/main/java/com/github/khanshoaib3/minecraft_access/features/PlayerWarnings.java [hunk 1]
```diff
@@ -68,48 +73,62 @@ private void loadConfigurations() {
...
-            MainClass.speakWithNarrator(I18n.translate("minecraft_access.player_warnings.health_low", health), true);
+            MainClass.speakWithNarrator(I18n.translate("minecraft_access.player_warnings.health_low", health, maxHealth), true);
             if (playSound) minecraftClient.player.playSound(SoundEvents.BLOCK_ANVIL_LAND, 1.0f, 1.0f);
         }
 
-        if (health < secondHealthThreshold && health > 0 && isHealthBelowFirstThreshold && !isHealthBelowSecondThreshold) {
-            isHealthBelowSecondThreshold = true;
-            MainClass.speakWithNarrator(I18n.translate("minecraft_access.player_warnings.health_low", health), true);
+        if (health <= secondHealthThreshold && health > 0 && isHealthBelowFirstThreshold && !isHealthBelowSecondThreshold) {
+            isHealthBelowSecondThreshold  =true;
```
Comment (sgmedev) [inline common/src/main/java/com/github/khanshoaib3/minecraft_access/features/PlayerWarnings.java]: Formatting: should be ` = `, not `  =`
```suggestion
            isHealthBelowSecondThreshold = true;
```

**Review by sgmedev — DISMISSED** [review]
LGTM!

**Comment by boholder** [comment]
Maybe merge it after the merging workflow is fixed?

**Review by boholder — DISMISSED** [review]

**Review by sgmedev — APPROVED** [review]

**Review by boholder — DISMISSED** [review]
I hope you don't forget to make changes according to: <https://github.com/minecraft-access/minecraft-access/pull/349#discussion_r1849943190>

**Pull request merged**